keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -146,7 +182,7 @@ configuration:
 > GET /.well-known/openid-configuration
 {
   "issuer": "https://dev.cluster.internal",
-  "jwks_uri": "https://dev.cluster.internal/serviceaccountkeys/v1",
+  "jwks_uri": "https://dev.cluster.internal/openid/v1/jwks",
   "authorization_endpoint": "urn:kubernetes:programmatic_authorization",
```

Comment:
the current PR does not include this field, even though it is required by spec; xref https://github.com/kubernetes/kubernetes/pull/80724/files#r369248302 and resolve the approach there, then update here if needed

Reply:
Removed from this KEP since we decided to omit it for now.